modules/claims_api/app/sidekiq/claims_api/v2/poa_form_builder_job.rb

```diff
@@ -16,7 +16,7 @@ class PoaFormBuilderJob < ClaimsApi::ServiceBase
       # it queues a job to update the POA code in BGS, as well.
       #
       # @param power_of_attorney_id [String] Unique identifier of the submitted POA
-      def perform(power_of_attorney_id, form_number, rep_id)
+      def perform(power_of_attorney_id, form_number, rep_id, icn_for_vanotify = nil)
```

Comment:
https://github.com/department-of-veterans-affairs/vets-api/pull/18687/files#r1781912411 same